src/ResourceManager/RecoveryServices.Backup/Cmdlets/ProtectionPolicy/GetAzureRmRecoveryServicesPolicy.cs

```diff
@@ -35,14 +35,17 @@ public class GetAzureRmRecoveryServicesProtectionPolicy : RecoveryServicesBackup
 
         public override void ExecuteCmdlet()
         {
-            base.ExecuteCmdlet();
+            ExecutionBlock(() =>
```

Comment:
I have some conflicting changes in policy cmdlets. Can you please revert your changes in policy cmdlets in your PR? I will do these changes in my next checkin.